spec/src/main/asciidoc/app-programming-model.adoc

```diff
@@ -196,6 +197,8 @@ The implementation must support the following annotation targets:
 NOTE: This annotation has changed in MicroProfile Metrics 2.0: Counters now always increase monotonically upon invocation.
 The old behaviour pre 2.0 can now be achieved with `@ConcurrentGauge`.
 
+If the metric no longer exists in the `MetricRegistry` when this annotation is invoked then an `IllegalStateException` will be thrown.
```

Comment:
I don't feel "annotation is invoked" is the right term to use, can we use "annotated element is invoked" instead?

Reply:
fixed!

Reply:
Woops, sorry now it's fixed ;)